subiquity/server/controllers/filesystem.py

```diff
@@ -205,48 +184,65 @@ def guided_lvm(self, target, mode=None, lvm_options=None):
                 mount="/",
                 ))
 
+    @functools.singledispatchmethod
+    def start_guided(self, target):
```

Comment:
I think a docstring and a return type annotation would be good here.

Reply:
Docstrings on the implementations would be nice too, perhaps.

Reply:
Done